src/main/java/Calculator.java

```diff
@@ -0,0 +1,83 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Calculator {
+    double sum;
+    Scanner scaner = new Scanner(System.in);
+    ArrayList<Product> products = new ArrayList<>();
+
+
+    private String IputName()
```

Comment:
🍏 В java принято именовать методы в стиле camelCase - с маленькой буквы:
```suggestion
    private String inputName()
```